tests/tls_apis/enc/enc.cpp

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/edger8r/enclave.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/report.h>
+#include <openenclave/internal/tests.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "tls_t.h"
+
+// This is the identity validation callback. A TLS connecting party (client or
+// server) can verify the passed in "identity" information to decide whether to
+// accept a connection reqest
+oe_result_t enclave_identity_verifier(oe_identity_t* identity, void* arg)
+{
+    oe_result_t result = OE_VERIFY_FAILED;
+
+    (void)arg;
+    OE_TRACE_INFO("enclave_identity_verifier is called with parsed report:\n");
+
+    // Check the enclave's security version
+    if (identity->security_version < 1)
+    {
+        OE_TRACE_ERROR(
+            "identity->security_version checking failed (%d)\n",
+            identity->security_version);
+        goto done;
+    }
+
+    // Dump an enclave's unique ID, signer ID and Product ID. They are
+    // MRENCLAVE, MRSIGNER and ISVPRODID for SGX enclaves In a real scenario,
```

Comment:
nit: Fix punctuation (add period after "enclaves").  It's correct in host.cpp but enc.cpp here doesn't match host.cpp.

Reply:
Fixed.